Makefile.am

```diff
@@ -20,3 +20,5 @@
 SUBDIRS = src scenes
 ACLOCAL_AMFLAGS = -I m4
 
+install-exec-hook:
+	echo -e "Snaskii has been installed, you can now start playing!"
```

Comment:
I know this might not be the point of this issue or PR, but this echo message is printed to the user. You can supress it by putting @ before echo, You also might try to supress the other make install messages if have time, but don't know if using @ will solve in this case.